python/cudf/cudf/core/indexed_frame.py

```diff
@@ -398,10 +398,18 @@ def sort_index(
                 inds = idx._get_sorted_inds(
                     ascending=ascending, na_position=na_position
                 )
-                # TODO: This line is abusing the fact that take accepts a
+                # TODO: This line is abusing the fact that _gather accepts a
                 # column, not just user-facing objects. We will want to
                 # refactor that in the future.
```

Comment:
I'm not 100% sure that this is relevant after this switch. We can leave it for now and investigate later, but this comment was necessary because `take` is a user-facing API that shouldn't accept columns (columns are internal). With `_gather`, which is an internal API, it may always make sense for it to accept columns.

Reply:
lol. Yes I blindly renamed it, looks like the comment can be removed?

Reply:
I think so, yeah.